### PR TITLE
Refined CONTRIBUTING with markdown syntax and steps

### PR DESCRIPTION
**Pull request by hellotunmbi**
I edited the CONTRIBUTING.md file with steps on how to contribute to this project.
Also, I made the update in proper markdown syntax

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
+- on Github, find and fork the source repository
+- On your computer, clone the fork repository
+- Commit your changes in a new branch;
+- Push your branch and submit a pull-request for it
+- Go through the review process until your pull-request is merged
+- Close the issue
 
-<p>Thanks for taking the time to contribute to this project. Before making PRs, please note the following:</p>
 
-<h3>Uploading new images</h3>
```
Comment (srbcheema1) [inline CONTRIBUTING.md]: please add this info also to your commit

**Review by srbcheema1 — COMMENTED** [review]
thanks for the refining of markdown syntax 
just add that info that i had mentioned about resolutions of pictures to be added . 
because many contributers were contributing wrong size of images .

**Comment by srbcheema1** [comment]
thanks for the PR @hellotunmbi 
there is some info missing .
fix that too.
i an eager to merge your PR :)

**Comment by hellotunmbi** [comment]
Okay @srbcheema1 I have added the resolutions info to my commit.

**Comment by srbcheema1** [comment]
thanks for the PR @hellotunmbi 
you have gone a good job :)

**Comment by hellotunmbi** [comment]
Thanks

**Pull request merged**